source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,61 @@ velit congue vel.
 D. Perform Encrypted Read/Write Operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Etiam hendrerit posuere justo, ac hendrerit lorem pellentesque sed.
-Fusce ullamcorper neque eu libero aliquam, sed placerat sapien cursus.
-Vestibulum maximus magna sed velit feugiat tempor.
+Read and write operations on encrypted fields use the same methods and have the capabilities of traditional MongoDB read and write operations with the added security of field-level encryption.
 
-Ut accumsan pellentesque metus, at porta felis dapibus vulputate. Aenean
-malesuada interdum libero in feugiat. Fusce pharetra massa eget leo
-varius, et gravida massa tempus.
+This section continues the sample guide of the MedcoMD Management System. Now that their schemas are defined, MedcoMD engineers are tasked with creating a system that allows doctors, who have access to the FLE Enabled client, to read and write to documents in the patient’s collection without restriction. Receptionists, who have access to the non-FLE enabled client, have restricted access to documents in the patient’s collection that prevents them from reading and writing to sensitive fields. 
+By following this section, you learn the difference in output when using a secure client with FLE enabled, and a client without FLE enabled.
+A patient is inserted into the patient’s collection using the following code. 
 
-Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
-tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
-in faucibus orci luctus et ultrices posuere cubilia Curae.
 
-Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
-erat.
+.. tabs::
+
+   tabs:
+
+     - id: java-jsonschema-generation
+       name: "Java"
+       content: |
+       
+        .. code-block:: java
+
+            public static void insertPatient(MongoCollection collection, String name, int ssn, String bloodType, ArrayList<Document> medicalRecords, int policyNumber, String provider) {
+                Document patient = new Document();
```

Comment:
Could you use the `append` pattern from the driver docs to construct this? The insertOne example shows how to create one with a subdocument too:
 https://mongodb.github.io/mongo-java-driver/3.11/driver/tutorials/perform-write-operations/

Reply:
Fixed: 

Switched put to append